config/default.yml

```diff
@@ -174,6 +174,18 @@ Bundler/GemComment:
   IgnoredGems: []
   OnlyFor: []
 
+Bundler/GemVersionDeclaration:
+  Description: 'Requires or prohibits gem version declarations.'
+  Enabled: false
+  VersionAdded: '1.13.0'
```

Comment:
Should this be left blank? `1.13.0` is already published. I wasn't sure what to put here.